edx/edx-enterprise
# ENT-278: Allow users with exotic characters in email addresses to be unlinked

**Description:** Previously, a failure to urlencode users' email addresses when submitting them to be unlinked from an enterprise customer meant that users with characters like `+` in their email addresses could not be unlinked. This pull request ensures that email addresses are urlencoded prior to their submission.

**JIRA:** [ENT-278](https://openedx.atlassian.net/browse/ENT-278)

**Dependencies:** None

**Merge deadline:** 10 April 2017

**Testing instructions:**

1. With the existing code base, link a user with an email address that includes `+`.
2. In the Django admin Manage Learners page, attempt to unlink that learner. Observe that it fails, and that the error message produced replaces the `+` with a ` `.
3. Pull this code.
4. Again attempt to unlink the same user.
5. Observe that the process succeeds.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
